src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -343,7 +343,7 @@ public async Task Put()
                         [nameof(DatabasePutResult.Name)] = databaseRecord.DatabaseName,
                         [nameof(DatabasePutResult.Topology)] = topology.ToJson(),
                         [nameof(DatabasePutResult.NodesAddedTo)] = nodeUrlsAddedTo,
-                        [nameof(DatabasePutResult.ShardsDefined)] = databaseRecord.Shards != null && databaseRecord.Shards.Length >= 2
+                        [nameof(DatabasePutResult.ShardsDefined)] = databaseRecord.Shards != null && databaseRecord.Shards.Length > 0
```

Comment:
We actually have property `IsSharded` on the database record.

Reply:
done